src/App.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
-import './App.css';
-import './index.js';
+import SquareWrapper from './SquareWrapper/SquareWrapper';
+import './index.css';
 
-//var arr = [square];
+class App extends React.Component {
+  render() {
+    return (
+      <div>
+        <SquareWrapper />
+        <div class='square'></div>
+      </div>
+    );
+  }
+}
```

Comment:
1. use FC instead of class

2. you didn't wrap your square, inspect your html to see what is the result